packages/edit-site/src/components/global-styles/screen-color-palette.js

```diff
@@ -18,9 +18,9 @@ function ScreenColorPalette( { name } ) {
 	return (
 		<>
 			<ScreenHeader
-				title={ __( 'Palette' ) }
+				title={ __( 'Edit Palette' ) }
```

Comment:
```suggestion
				title={ __( 'Edit palette' ) }
```

I don't have a strong opinion about changing the text, but would it be better to unify it with sentence cases? (example
`Edit template`, `Edit styles`, etc.)